drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/projects_importer.rb

```diff
@@ -0,0 +1,222 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis::Importers
+  class ProjectsImporter
+    include NotifierConfig
+
+    AbortImportException = Class.new(StandardError)
+
+    attr_accessor :attempt
+    attr_accessor :data_source
+    attr_accessor :dir
+
+    def initialize(dir:, key:, etag:)
+      self.attempt = ProjectsImportAttempt.where(etag: etag, key: key).first_or_initialize
+      self.data_source = HmisExternalApis::AcHmis.data_source
+      self.dir = dir
+    end
+
+    def run!
+      start
+      sanity_check
+      # validate
+      ProjectsImportAttempt.transaction do
+        upsert_funders
+        upsert_orgs
+        upsert_projects
+        upsert_walkins
+      end
+      analyze
+      finish
+    rescue AbortImportException => e
+      # FIXME: see if this is right
+      @notifier.ping(
+        'Failure in project importer',
+        {
+          exception: e,
+          info: nil,
+        },
+      )
+      Rails.logger.fatal e.message
+    end
+
+    def start
+      setup_notifier('HMIS Projects')
+      Rails.logger.info "Starting #{attempt.key}"
+      attempt.attempted_at = Time.now
```

Comment:
```suggestion
      attempt.attempted_at = Time.current
```
I think Theron is throwing in a rubocop for that, but might as well fix them now.